src/tree/GitHubConfigGroupTreeItem.ts

```diff
@@ -69,7 +100,18 @@ export class GitHubConfigGroupTreeItem extends AzExtParentTreeItem {
         return false;
     }
 
+    public get errorRange(): Range | undefined {
+        return this.parseYamlError instanceof YAMLSyntaxError ? getRangeFromError(this.parseYamlError) : undefined;
+    }
+
     public async loadMoreChildrenImpl(): Promise<AzExtTreeItem[]> {
+        if (this.errorRange) {
```

Comment:
You should use `this.parseYamlError` as the check since that's what you're using everywhere else.